vnext/Microsoft.ReactNative/Views/DevMenu.cpp

```diff
@@ -106,6 +111,24 @@ void DevMenuManager::CreateAndShowUI() noexcept {
       Mso::React::ReactOptions::UseFastRefresh(m_context->Properties()) ? L"Disable Fast Refresh"
                                                                         : L"Enable Fast Refresh");
 
+  devMenu.SamplingProfilerText().Text(
+      !Microsoft::ReactNative::HermesSamplingProfiler::IsStarted() ? L"Start Hermes sampling profiler"
+                                                                   : L"Stop Hermes sampling profiler");
+  std::ostringstream os;
+  if (Microsoft::ReactNative::HermesSamplingProfiler::IsStarted()) {
+    os << "Sampling profiler running..";
+  } else {
+    os << "Click to start.";
+  }
+
+  auto lastTraceFilePath = Microsoft::ReactNative::HermesSamplingProfiler::GetLastTraceFilePath();
+  if (!lastTraceFilePath.empty()) {
+    os << std::endl << "Samples from last invocation are stored at " << lastTraceFilePath.c_str() << "  (path copied to clipboard).";
+    os << std::endl << "Navigate to \"chrome:\\tracing\" and load the trace file.";
```

Comment:
should use `edge://tracing` instead :)

Reply:
Are we finally ready to assume people have edge? -- Should we update the dependencies script to check for edge instead of chrome now?